packages/markdown/component/readme.md

```diff
@@ -3,15 +3,14 @@
 This package brings legacy support for the `<Markdown />` component to all Astro projects.
 
 > The `<Markdown />` component does not work in SSR. Consider [importing Markdown content](https://docs.astro.build/en/guides/markdown-content/#importing-markdown) instead.
-:::
+> :::
```

Comment:
This `:::` looks like it got copied over from our docs syntax at some point, and should probably be removed:

```suggestion
```